motioneye/locale/de/LC_MESSAGES/motioneye.po

```diff
@@ -322,182 +285,159 @@ msgstr "Kamera Name …"
 msgstr "Geben Sie eine benutzerdefinierte Auflösungshöhe ein"
 
 #: motioneye/templates/main.html:308
-#, fuzzy
 msgid "Video-Rotacio"
 msgstr "Videodrehung"
 
 #: motioneye/templates/main.html:317
-#, fuzzy
 msgid "uzu ĉi tion por turni la kaptitan bildon, se via kamerao ne estas poziciigita ĝuste"
-msgstr "Verwenden Sie diese Option, um das aufgenommene Bild zu drehen, wenn Ihre Kamera nicht richtig positioniert ist"
+msgstr ""
+"Verwenden Sie diese Option, um das Bild zu drehen, wenn Ihre Kamera nicht "
+"richtig positioniert ist"
 
 #: motioneye/templates/main.html:320
-#, fuzzy
 msgid "Kadrofrekvenco"
-msgstr "Bildrate"
+msgstr "Framerate"
 
 #: motioneye/templates/main.html:322
-#, fuzzy
 msgid "agordas la nombron de kadroj kaptitaj de la kamerao ĉiun sekundon (pli altaj valoroj produktas pli mildajn filmetojn sed postulas pli da CPU-potenco, pli grandan stokadon kaj larĝan bandon)"
 msgstr ""
 "Legt die Anzahl der von der Kamera pro Sekunde aufgenommenen Bilder fest ("
 "höhere Werte erzeugen flüssigere Videos, erfordern jedoch mehr CPU-Leistung, "
 "mehr Speicher und Bandbreite)"
 
 #: motioneye/templates/main.html:328
-#, fuzzy
 msgid "Privateca masko"
 msgstr "Datenschutzmaske"
 
 #: motioneye/templates/main.html:330
-#, fuzzy
 msgid "ebligas maskeradon de bildoj por eviti registradon de iuj bildaj areoj por protekti privatecon"
-msgstr "Ermöglicht das Maskieren von Bildern, um das Aufzeichnen bestimmter Bildbereiche zum Schutz der Privatsphäre zu vermeiden"
+msgstr ""
+"Ermöglicht das ausblenden von Bereichen im Bild, um das Aufzeichnen "
+"bestimmter Bildbereiche zum Schutz der Privatsphäre zu vermeiden"
 
 #: motioneye/templates/main.html:335 motioneye/templates/main.html:996
 #, fuzzy
 msgid "Redakti maskon"
 msgstr "Maske bearbeiten"
 
 #: motioneye/templates/main.html:336 motioneye/templates/main.html:997
-#, fuzzy
 msgid "Konservi maskon"
-msgstr "Behalte eine Maske"
+msgstr "Maske Speichern"
 
 #: motioneye/templates/main.html:339
 #, fuzzy
 msgid "alklaku ĉi tiun butonon por ebligi/malebligi la maskan redaktilon"
 msgstr "Klicken Sie auf diese Schaltfläche, um den Maskeneditor zu aktivieren / deaktivieren"
 
 #: motioneye/templates/main.html:350
-#, fuzzy
 msgid "Ekstraj Elektoj"
 msgstr "Zusätzliche Optionen"
 
 #: motioneye/templates/main.html:352
-#, fuzzy
 msgid "vi povas aldoni iujn ajn pliajn opciojn ĉi tie por la «motion» demono (uzu la «nomo valoro» formaton, unu opcion por linio)"
 msgstr ""
-"Sie können hier zusätzliche Optionen für den \"Motion\" -Dämon hinzufügen ("
-"verwenden Sie das Format \"Name Value\", eine Option pro Zeile)"
+"Sie können hier zusätzliche Optionen für den \"Motion\"-Dienst hinzufügen ("
+"verwenden Sie das Format \"Name Wert\" und eine Option pro Zeile)"
 
 #: motioneye/templates/main.html:361
-#, fuzzy
 msgid "elektu kie kaj kiel viaj amaskomunikiloj estas konservitaj"
 msgstr "Wählen Sie, wo und wie Ihre Medien gespeichert werden"
 
 #: motioneye/templates/main.html:362
-#, fuzzy
```

Comment:
@jmichault 
Should we consequently keep the fuzzy flag do that translations do not break when code lines are added or removed? Or does it work regardless?

Reply:
The fuzzy flag marks the sentences to be corrected. It is deleted when the sentence is translated. It does not influence the translation.